packages/logs/src/boot/logsPublicApi.ts

```diff
@@ -91,13 +91,7 @@ export function makeLogsPublicApi(startLogsImpl: StartLogs) {
     removeLoggerGlobalContext: monitor(globalContextManager.remove),
 
     createLogger: monitor((name: string, conf: LoggerConfiguration = {}) => {
-      customLoggers[name] = new Logger(
-        (...params) => handleLogStrategy(...params),
-        name,
-        conf.handler,
-        conf.level,
-        conf.context
-      )
+      customLoggers[name] = new Logger(handleLogStrategy, name, conf.handler, conf.level, conf.context)
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**❗ warning:** FMU using `handleLogStrategy` directly here should break loggers created before the init 🤔 
do we have a test for this one?‏

Reply:
Indeed. I fixed it and added a test.